static/app/views/discover/eventDetails/index.spec.tsx

```diff
@@ -280,20 +303,24 @@ describe('Discover > EventDetails', function () {
       router: {
         location: {
           pathname: '/organizations/org-slug/discover/project-slug:deadbeef',
-          query: {...allEventsView.generateQueryStringObject(), homepage: true},
+          query: {...allEventsView.generateQueryStringObject(), homepage: '1'},
         },
       },
     });
 
     render(
       <EventDetails
+        {...TestStubs.routeComponentProps()}
         organization={organization}
         params={{eventSlug: 'project-slug:deadbeef'}}
         location={router.location}
       />,
       {context: routerContext, organization}
     );
 
+    // @TODO this seems wrong, pathname is not a valid html attribute - we are likely
+    // bleeding props into the dom here...
+    // @ts-expect-error
     expect((await screen.findByText('Discover')).pathname).toEqual(
```

Comment:
cc @narsaynorath because I saw you added this test

Reply:
You're right, this is weird. Can we change it to this?

```js
    const breadcrumb = await screen.findByTestId('breadcrumb-link');
    expect(breadcrumb).toHaveTextContent('Discover');
    expect(breadcrumb).toHaveAttribute(
      'href',
      expect.stringMatching(new RegExp('^/organizations/org-slug/discover/homepage/?'))
    );
```